content/en/synthetics/private_locations/_index.md

```diff
@@ -188,6 +176,10 @@ docker run --rm -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetic
 
 This command starts a Docker container and makes your private location ready to run tests. **Datadog recommends running the container in detached mode with proper restart policy.**
 
+<div class="alert alert-info">You can upload custom root certificates to your private locations to have your API and browser tests perform SSL handshake using your own `.pem` files with the <a href="https://hub.docker.com/r/datadog/synthetics-private-location-worker">Synthetics Private Location Worker</a> version 1.5.3 or later.</div>
```

Comment:
why is it in a "info" ?

Reply:
Want to do red (danger) or yellow (warning) instead?